frontend/src/views/account-mgmt/FacilityView.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <v-container fluid>
+    <h1>Update Facility Information</h1>
+    <v-row>
+      <v-col class="ml-6 mt-6 pb-0">
+        <h4>Facility Details</h4>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col class="ml-6 pt-0 pb-0">
+        <FacilityInfo :facilityId="facilityId" class="mt-0" />
```

Comment:
In FacilityInfo do we want a <v-divider /> above Physical Address (to be consistent with Organization View)? Note, this would only apply when the Facility Name/edit buttons are available.

Also, In FacilityInfo should we use an editable prop instead of isApplicationPage? This is what we did for OrganizationInfo.

Reply:
re: "In FacilityInfo do we want a above Physical Address (to be consistent with Organization View)? Note, this would only apply when the Facility Name/edit buttons are available." Do you mean the horizontal line above phys address in org is missing where there is one in facilites?

re:  "Also, In FacilityInfo should we use an editable prop instead of isApplicationPage?... " i can do this, will have to carry this work into the weekend, which i'm fine with.

Reply:
Yes, I mean the horizontal line. Let's tackle these updates when you do Edit.